packages/E2ETest/run_wdio.js

```diff
@@ -100,20 +114,42 @@ function parseLogs() {
   return names;
 }
 
+function PrintFailedTests(ft) {
+  for (let i = 0; i < Object.keys(ft).length; i++) {
+    const key = Object.keys(ft)[i];
+    console.log(chalk.redBright(key));
+    console.log(
+      '  ',
+      chalk.underline('testcase'),
+      chalk.bold(ft[key].testcase)
```

Comment:
Looks like this is missing a space. Template strings are help to avoid this. Same on L126.

Reply:
since I have different formatting, this feels cleaner to me